docs/_sources/week02/display-reports/index.rst.txt

```diff
@@ -0,0 +1,158 @@
+:orphan:
+
+.. _week2_display-reports:
+
+==========================================
+Project Week2: Display Zika Reports on Map
+==========================================
+
+We have an empty interactive map from OSM.
+
+We need to create a new Layer in OpenLayers that will represent our Zika report data.
+
+This will require a fair amount of work, luckily it is primarly things we did in our `Airwaze project <../../studios/airwaze/>`_.
+
+Tasks:
+    - Design Report.java class
+    - Create ReportTests.java
+    - Report table is populated from CSVs
+    - Create ReportTests.java
+    - Create ReportController.java
+    - Create ReportControllerTests.java
+    - Create ReportRepository.java
+    - Create ReportRepositoryTests.java
+    - Amend script.js
+    - Verify Zika report layer renders
+    - Commit
+
+Design Report.java Class
+------------------------
+
+Before we display our report data as a layer on our map in OSM we will first have to transform our data into a state we can work with.
+
+Ultimately OpenLayers is expecting a source of data in a GeoJSON format. Right now our data is locked away in various CSV files.
+
+Let's consider our data's journey.
+
+It currently exists in CSV format, we will have to COPY our CSVs into a PostGIS table as records, our web application will convert records from PostGIS into a Java Object, our web app will convert Java Objects into GeoJSON. The GeoJSON is what OpenLayers is expecting to make a new layer from.
```

Comment:
What do you think about making this a bulleted or numbered list?

Reply:
I agree. A numbered list will help visualize the journey a little better.